### DISCUSSION TITLE
Best email provider to work with SimpleLogin

### DISCUSSION THREAD

**Discussion by EricGuic** [simple-login/app#464] [Q&A]
Hi,
As a "simple" user (not a developper), self-hosting my emails seems too complicated for now.
So I'm looking to the **best email provider** that respects my private life, and will work nicely alongside **SimpleLogin** (and why not one which has implemented SimpleLogin directly in his offer, if that exists ?).

I've seen ProntonMail used by you Son Nguyen Kim in your tutorial video (but ProntonMail begins to receive more and more negative reviews from users, mainly because of a slow implementation of the features they promised a long time ago).
Is that a real issue for you, or ProtonMail is a good option ?

If not, which other email provider would you recommend (and if that matters, for user here in France/Europe) ?
Which one supports the best security features (DCKIM, DMARC and other specs I've found in SimpleLogin settings) ?

It could be a free or paid service, like :
- infomaniak (https://www.infomaniak.com/fr/email-gratuit)
- gandi
- ovh

Thanks for your advice to help a standard user free himself from GAFAM ;-)

Best regards from France.

**Comment by nguyenkims** [comment]
Salut Eric! In the video I used ProtonMail as it's used by a lot of SimpleLogin users. SimpleLogin should work with any email service and we try to not favor an email service over another :). Here's 2 configurations that I often see:

- Use a popular email service (Gmail/Outlook/...) along with a PGP software. There's a lot of options for PGP software on https://www.openpgp.org/software/
- use ProtonMail and their default PGP support. ProtonMail doesn't directly support IMAP as far as I know and requires a "bridge" in order to use an email client as Thunderbird. 

Personally I use both :).

**Reply by EricGuic** [reply]
Thanks for your quick and full of details reply. I will explore this.
Best regards, Eric.
